cypress/utils/utils.ts

```diff
@@ -367,60 +351,52 @@ export function applySelectFilter(filterId, filterName, filterText, isValid = tr
 
 export function applySearchFilter(
     filterName: string,
-    searchText: any,
-    identifiedRisk?: boolean,
+    searchText: string | string[],
+    identifiedRisk: boolean = false,
```

Comment:
No need to put type if you assign value here, type is defined automatically. Just a little tip...

Reply:
Yesss you're right, I'' change it, thanks!

Reply:
Changed in 9436e1e6640e32e79394db91d7ebd6701ee11710